docs/examples/search-external-access.md

```diff
@@ -0,0 +1,72 @@
+# Enable Alfresco Search Services (`/solr`) External Access
+This example demonstrates how to enable Alfresco Search services for external access.
```

Comment:
Please write the product name as 'Alfresco Search Services' throughout the doc (may affect other files in this PR)